front/src/common/Map/Buttons/MapButtons.tsx

```diff
@@ -97,14 +106,20 @@ export default function MapButtons({
 
   return (
     <div ref={mapButtonsRef}>
-      <div className="btn-map-container">
+      <div
+        className={cx({
+          'btn-map-container': !isInEditor,
+          'btn-map-container-editor': isInEditor,
+        })}
```

Comment:
See next css comment

```suggestion
      className={cx('btn-map-container', {
          editor: isInEditor,
        })}
```